project/HaKkaz/README.md

```diff
@@ -0,0 +1,67 @@
+NSD Term Project Proposal
+---
+
+## Basic Information
+GitHub repository: [Point in Polygon](https://github.com/HaKkaz/Point-in-Polygon)
+
+The Point in Polygon problem is a fundamental problem in computational geometry that involves determining whether a given point lies inside, outside, or on the boundary of a polygon.
```

Comment:
One line is used as a paragraph and it is too long.  It is hard to add comments at specific location for a long line.  Please change to 79-char width.